### PR TITLE
Request to merge - Calendar Changes to cross browsers

### PR DESCRIPTION
**Pull request by vazmymo**
Desktop and Mobile browser combinations tested

### CODE REVIEW

**Comment by umahaea** [comment]
@vazmymo - I notice that you have commented all the tests except Calendar. For PR, we need to make sure we are in sync with the master.Please go ahead and uncomment the other tests. Rebase it once again with master and send over the PR.

File: src/main/java/elements/elementsPageObjects/CalendarPageObjects.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+	public By iconRight=By.id("icon_right");
+	public By daySunday=By.id("day_sunday");
+	public By dayMonday=By.id("day_monday");
+	public By dayTuesday=By.id("day_tuesday");
+	public By dayWednesday=By.id("day_wednesday");
+	public By dayThursday=By.id("day_thursday");
+	public By dayFriday=By.id("day_friday");
+	public By daySaturday=By.id("day_saturday");
+	
+
```
Comment (umahaea) [inline src/main/java/elements/elementsPageObjects/CalendarPageObjects.java]: Could you cut down these extra line spaces?

File: src/test/java/elementsTests/CalendarTest.java [hunk 2]
```diff
@@ -0,0 +1,1052 @@
...
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+
+import utilities.BaseClass;
+
+public class CalendarTest extends BaseClass {
```
Comment (umahaea) [inline src/test/java/elementsTests/CalendarTest.java]: could you format this class?

File: testng.xml [hunk 3]
```diff
@@ -7,23 +7,24 @@
...
+    <parameter name="mobDeviceName" value="Google Nexus 7 HD Emulator"></parameter><!-- Android or iOS : iPhone 6 Plus/iPad Air/ iPad Pro -->
+    <parameter name="mobilePlatformVer" value="4.4"></parameter><!-- mobile OS version -->
+    <parameter name="mobBrowser" value="Browser"></parameter><!-- browser eg.safari, chrome -->
     <parameter name="appiumVer" value="1.5.2"></parameter><!-- version of appium on sauce -->
 
     <test name="ElementsTests">
         <groups>
             <run>
                 <include name="desktop"/>
+                <exclude name="TypographyTest"/>
```
Comment (umahaea) [inline testng.xml]: Remove this "exclude" tag.

**Comment by vazmymo** [comment]
Calendar Formatting Issues Fixed

**Comment by umahaea** [comment]
Thanks @vazmymo this looks good to me.. Still not bit convinced with the formatting. Let me do it this time from my machine. We can merge this change only after https://github.com/Pearson-Higher-Ed/elements/pull/82 is merged. Our tests in "master" branch should always point to 'V0' branch of elements. So I would like to hold off on this until we see Calendar on elements V0.

**Comment by umahaea** [comment]
Now the calendar changes are available on V0 elements, I'm merging the calendar tests to master.

**Pull request merged**